src/foam/u2/view/PasswordView.js

```diff
@@ -18,7 +18,77 @@
 foam.CLASS({
   package: 'foam.u2.view',
   name: 'PasswordView',
-  extends: 'foam.u2.tag.Input',
+  extends: 'foam.u2.View',
 
-  properties: [ [ 'type', 'password' ] ]
```

Comment:
We still need this for the `type` attribute on the HTML input element to be set correctly.

```JavaScript
{
  class: 'String',
  name: 'type',
  value: 'password'
}
```